BRANDING.md

```diff
@@ -0,0 +1,55 @@
+
+## Purpose of the Brand Guidelines
+    
+These guidelines exist to ensure that the Istio project can share its platform under open source licenses while making sure that the “Istio” brand is protected as a meaningful source identifier in a way that is consistent with trademark law.
+ 
+## Acceptable Uses
+
+Given the open nature of Istio, you may use the Istio trademark to refer to the project without prior written permission. Examples of these acceptable references include the following:
+
+-   To refer to the Istio project itself;
+    
+-   To refer to unmodified source code or other files shared by the Istio repositories on GitHub;
+    
+-   To accurately identify that your design or implementation is based on or is for use with the Istio platform.
+    
+-   To discuss the Istio project for educational or informational purposes
+    
+*Examples:*
+
+-   “[Your Product/Offering] for Istio.”
+    
+-   “[XYZ] Conference for Istio Users.”
+    
+-   “Making [XYZ] better with Istio”
+    
+## Usage Guidelines
+
+-   The Istio name may not be used or registered in a manner that would cause confusion as to the Istio project’s affiliation or endorsement.
+    
+-   Don’t use the Istio name as part of your company name, product name, domain name, or social media profile.
+    
+-   Other than as permitted by these guidelines, the Istio name should not be combined with other trademarks, terms, or source identifiers.
+    
+-   Don't remove, distort or alter any element of the Istio Trademarks without permission:
+    
+-   Do not shorten, abbreviate, or change the spelling of the Istio name, for example in mass communication, product names, social media profiles, domain names or event names).
+    
+-   Do not change the color or design elements of the Istio logo, or combine the logo with any other text.
+    
+-   Community websites, social media accounts, or other community-organized properties or organizations should follow the naming convention "[XYZ] for Istio Users" or "[XYZ] for Istio Contributors".
+    
+-   The name "Istio Working Group" can only be used by approved working groups maintaining code in the Istio GitHub organization under Istio Project governance.
+    
+-   If you’re organizing a conference, training, or certification program, you can use the word Istio to refer to the technology that the event or program covers. However, Istio cannot be contained in the title name for your conference or program (such as IstioConf or Certified Istio Administrator) without further explanation. An example of an appropriate usage could be “XYZ Conference for Istio Users” or “XYZ Certification Program for Istio Users” with XYZ being your own brand name. Any related materials should also prominently disclaim any official affiliation with or endorsement by the Istio project.
```

Comment:
After the 2nd sentence (before the example), perhaps add:
```
It is not acceptable to imply that the Istio community endorses the activity, or that the activity is officially
part of the Istio community, without explicit permission.
```

Reply:
The very first sentence in this area (currently line 32) now reads:

The Istio name may not be used or registered in a manner that would cause confusion as to the Istio project’s affiliation or endorsement.

I think that covers this. Feel free to re-comment if you disagree, Doug.